earth_enterprise/src/fusion/autoingest/MiscConfigStorage.idl

```diff
@@ -40,6 +40,9 @@ class MiscConfigStorage {
   uint         PackGenAnalysisThreads = uint(1);
   GraphOpsType GraphOperations = MiscConfigStorage::NO_GRAPH_OPS;
 
+  // True to minify previous imagery level, false to minify source image
```

Comment:
A few of points about this comment:
- "True" will minify previous imagery and terrain levels, correct (not just imagery)?
- The "false" text is a bit confusing to me. I would say something like "false to build from source imagery"